src/commands/commands.h

```diff
@@ -36,6 +38,7 @@ void Graph_Profile(void *args);
 void Graph_Explain(void *args);
 int Graph_List(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Debug(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
+int Graph_Info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Delete(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
```

Comment:
Try to keep lines "sorted" by length:
```c
void Graph_Explain(void *args);
 int Graph_List(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Debug(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
 int Graph_Delete(RedisModuleCtx *ctx, RedisModuleString **argv, int argc);
```

Reply:
This is done out of pure esthetics.
It is a habit I wish everyone in the team will follow, as I don't see this being enforced by an auto formatter.